.github/workflows/mdchecks.yml

```diff
@@ -0,0 +1,79 @@
+name: Markdown Checks
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths: '**.md'
+
+jobs:
+  mdchecks:
+    name: Markdown Checks
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout the Repo 
+      uses: actions/checkout@v2
+
+    - name: Set up NodeJS
+      uses: actions/setup-node@v1
+
+    - name: Get MD files in this PR
+      uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        # Paths to files will be available in `filter_files` output variable.
+        list-files: shell
+        filters: |
+          markdown:
+            - added|modified: '*.md'
+
+      # This step filters the list of files further, removing from consideration
+      # any file that includes the XML comment
+      # `<!-- BSSW_GITHUB_CI_IGNORE_MD_CHECKS -->` beginning on a first column
+      # anywhere in the file. The resulting list of files is available in
+      # `keep_files`.
+    - name: Skip Ignore Files
+      id: keep
+      run: |
+        mdfiles=""
+        for f in ${{ steps.filter.outputs.markdown_files }}; do 
+          if [ -n "$(grep '^<!-- BSSW_GITHUB_CI_IGNORE_MD_CHECKS -->' $f)" ]; then
+            continue
+          fi
+          mdfiles="$mdfiles $f"
+        done
+        echo ::set-output name=keep_files::${mdfiles}
```

Comment:
This should be partitioned into a script maintained in the Git repo called something like `filter_markdown_files_to_check.sh` and then call it something like:

```
      run: |
        <base-dir>/filter_markdown_files_to_check.sh ${{ steps.filter.outputs.markdown_files }}
```